online-shop/src/app/types/createOrderDto.ts

```diff
@@ -0,0 +1,8 @@
+import { OrderDetailDto } from "./orderDetailDto";
+
+export interface CreateOrderDto{
```

Comment:
I think it's a bit weird to name stuff DTO in Angular. The frontend doesn't really know what a DTO is. I would just remove that from the name.